<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsClassClass</class>
 <widget class="QMainWindow" name="QtWidgetsClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome to your Guide to travel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Get Started</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>871</width>
        <height>481</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../bg.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>label_3</zorder>
     <zorder>pushButton</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>1</y>
        <width>531</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>131</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Traverse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_15">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>All Paths</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_18">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Connected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_20">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Complete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>←</string>
        </property>
        <property name="icon">
         <iconset theme="arrow"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>131</width>
          <height>428</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Source City</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Destination City</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_3"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Transportation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_4"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>380</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>428</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Source City</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_9"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Destination City</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_10"/>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Tansportation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_11"/>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>New Price</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_12"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>←</string>
        </property>
        <property name="icon">
         <iconset theme="arrow"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>380</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>←</string>
        </property>
        <property name="icon">
         <iconset theme="arrow"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>131</width>
          <height>319</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Source City</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_13"/>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Destination City</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_14"/>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Tansportation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_15"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>←</string>
        </property>
        <property name="icon">
         <iconset theme="arrow"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Traverse</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_16">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>112</y>
          <width>129</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>129</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Source City</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>DFS</string>
        </property>
        <property name="icon">
         <iconset theme="arrow"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_14">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>BFS</string>
        </property>
        <property name="icon">
         <iconset theme="arrow"/>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>131</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>30</y>
        <width>131</width>
        <height>210</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Source City</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_5"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Destination City</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_6"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>380</y>
        <width>129</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>210</y>
        <width>129</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>180</y>
        <width>129</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Transportations </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>131</width>
        <height>319</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Source City</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_17"/>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Destination City</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_18"/>
       </item>
       <item>
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Budget</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_19"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string>←</string>
      </property>
      <property name="icon">
       <iconset theme="arrow"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>All Paths</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>441</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
